include/google/protobuf/any.proto

```diff
@@ -0,0 +1,158 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
I didn't fully get why we have all of these proto files, would you mind expanding?

Reply:
Hi @pierDipi these files got in by running `./hack/update-codegen.sh`. They're probably not related to this PR. Let me rebase and see if this is still required.

Reply:
OK. The codegen check now passes without those files. So, this PR doesn't include them anymore.